#request
bool request

---

#response

geometry_msgs/Point32[] points